core/src/main/java/com/linecorp/armeria/server/healthcheck/HealthCheckServiceBuilder.java

```diff
@@ -47,12 +47,26 @@ public final class HealthCheckServiceBuilder implements TransientServiceBuilder
     private static final double DEFAULT_LONG_POLLING_TIMEOUT_JITTER_RATE = 0.2;
 
     private final ImmutableSet.Builder<HealthChecker> healthCheckers = ImmutableSet.builder();
-    private AggregatedHttpResponse healthyResponse = AggregatedHttpResponse.of(HttpStatus.OK,
-                                                                               MediaType.JSON_UTF_8,
-                                                                               "{\"healthy\":true}");
-    private AggregatedHttpResponse unhealthyResponse = AggregatedHttpResponse.of(HttpStatus.SERVICE_UNAVAILABLE,
-                                                                                 MediaType.JSON_UTF_8,
-                                                                                 "{\"healthy\":false}");
+    private AggregatedHttpResponse healthyResponse =
+            AggregatedHttpResponse.of(HttpStatus.OK, MediaType.JSON_UTF_8,
+                                      "{\"healthy\":true,\"status\":\"HEALTHY\"}");
```

Comment:
Note: In case this point was missed, armeria client is ignoring the body at the moment and determining health status via status code.

Reply:
Oh right. `HttpHealthChecker` [only checks the status code](https://github.com/line/armeria/blob/fea0622618613631735b4625c3bf8faf20d9f87a/core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthChecker.java#L211). Should we handle the changes in this PR as well?

Reply:
I'm unsure of how we want to expose health check information on the client-side at the moment. From an xDS perspective, other signals such as 1) the ratio of failed health checks 2) time since last health check 3) etc.. are exposed.

I think it's better to focus on implementing backwards compatible server-side changes in this PR, and client-side changes are dealt with separately (after a consensus is made on the API direction).